Automattic/wp-calypso
# Blogger Experience: add tags education link under Tags section

#### Changes proposed in this Pull Request

* According to [Tag education in the Editor](https://github.com/Automattic/wp-calypso/issues/55300), add "Build your audience with tags" to let bloggers learn more about how to tag their posts to attract new readers. As we cannot add education link to Gutenberg directly (see: https://github.com/WordPress/gutenberg/pull/33998), we have to use `addFilter` to inject the education link into Tags section. See more detail: [Custom Taxonomy Selector
 ](https://github.com/WordPress/gutenberg/tree/trunk/packages/editor/src/components/post-taxonomies#custom-taxonomy-selector)

##### Tags

![](https://user-images.githubusercontent.com/13596067/129162940-e72a1ec2-d25f-4c8b-83c2-11bfce6364c9.png)

##### Suggestion: Add tags

![](https://user-images.githubusercontent.com/13596067/129162983-30a46a76-aec0-4981-a7ee-7ddb91182329.png)

##### Events

![](https://user-images.githubusercontent.com/13596067/129163038-6d6a0bff-bff4-435a-803c-2dfe92e00614.png)

#### Testing instructions



* Sandbox your API
* Open another terminal and run cd apps/wpcom-block-editor && yarn dev --sync
* Open your local calypso
* Create or edit the post
* _**Check "Build your audience with tags" show on `Tags` section**_
* Click Publish
* _**Check "Build your audience with tags" show on `Suggestion: Add tags` section**_
* Turn on debugging by running localStorage.setItem( 'debug', 'wpcom-block-editor:*' ) within dev tools Console and reload the page
* Click "Build your audience with tags" in the Tags section
* **_Check dev tools Console for wpcom-block-editor:analytics:tracks Recording event "wpcom_block_editor_tags_education_link_click"_**



Related to https://github.com/Automattic/wp-calypso/issues/55300, https://github.com/Automattic/wp-calypso/pull/55375, https://github.com/WordPress/gutenberg/pull/33998
